README.md

```diff
@@ -38,40 +38,68 @@ The Vantage Discovery Python SDK is divided into several modules, allowing you t
 - __Search__: Perform semantic, embedding and "more like this/these" searches within your collections.
 - __LLM Keys Management__: Keep your LLM provider secrets safe and up-to-date.
 
-## Examples
+## 🔍 Examples
 
 ### Creating a Collection
 
+To create a new collection for storing documents, specify the collection ID, the dimension of the embeddings, and the LLM (language learning model) details. Here, we use `text-embedding-ada-002` from OpenAI with the necessary secret key. Visit our [documentation](https://docs.vantagediscovery.com/docs/management-api) for more options and details described.
```

Comment:
Instead of :

Visit our [documentation](https://docs.vantagediscovery.com/docs/management-api) for more options and details described.

I would go with:

More details in [management-api](https://docs.vantagediscovery.com/docs/management-api) documentation.

And, for the third example:

More details in [search-api](https://docs.vantagediscovery.com/docs/search-api) documentation.

When markdown is rendered, it looks like links are pointing at the same documentation in all three examples. It feels less formal and less copy/paste this way, at least to me.

Reply:
Yes, makes sense! Thank you